src/server/scripts/OutdoorPvP/OutdoorPvPNA.cpp

```diff
@@ -533,12 +574,38 @@ int32 OPvPCapturePointNA::HandleOpenGo(Player* player, GameObject* go)
 
 bool OPvPCapturePointNA::Update(uint32 diff)
 {
-    // let the controlling faction advance in phase
-    bool capturable = false;
-    if (m_ControllingFaction == TEAM_ALLIANCE && m_activePlayers[0].size() > m_activePlayers[1].size())
-        capturable = true;
-    else if (m_ControllingFaction == TEAM_HORDE && m_activePlayers[0].size() < m_activePlayers[1].size())
-        capturable = true;
+    if (!m_capturePoint)
+        return false;
+
+    float radius = ((float)m_capturePoint->GetGOInfo()->capturePoint.radius) - 60.0f;
+
+    for (uint32 team = 0; team < 2; ++team)
+    {
+        for (PlayerSet::iterator itr = m_activePlayers[team].begin(); itr != m_activePlayers[team].end();)
```

Comment:
You could use a range-based for loop here instead of iterators.

Reply:
Dunno how to do that, i just used the method that is used in the normal OutdoorPvP